src/stylesheets/blocks/_problem.scss

```diff
@@ -1,4 +1,94 @@
 .problem-section {
   background-color: color(mint-green);
+
+  @media (screen(lg)) {
+    padding-bottom: rem(75);
+  }
+
+  .-lg-width {
+    background-color: color(white);
+    padding-top: rem(40);
+    padding-bottom: rem(20);
+
+    @media (screen(lg)) {
+      border-radius: rem(4);
+    }
+  }
+
+  .lg-text {
+    color: color(warm-charcoal);
+
+    @media (screen(lg)) {
+      text-align: left;
```

Comment:
Text alignment must be set somewhere else in order for it to need to be reset back to `left` here. I see `.lg-text` has `text-align: center`. Might want to move that part to your section containers themselves then you won't have to reset them back to left in quite as many places.